modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsController.scala

```diff
@@ -254,21 +254,13 @@ object TcsController {
     implicit val eq: Eq[LightSource] =  Eq.fromUniversalEquals
   }
 
-  /** Data type for science fold position. */
-  sealed trait ScienceFoldPosition
-  object ScienceFoldPosition {
-    case object Parked extends ScienceFoldPosition
-    final case class Position(source: LightSource, sink: LightSinkName) extends ScienceFoldPosition
+  /* Data type for science fold position. */
+  final case class LightPath(source: LightSource, sink: LightSinkName)
```

Comment:
This type replaces `ScienceFoldPosition`. The problem with `ScienceFoldPosition` is that it tried to represent the light path request and the science fold position, and failed at both. It couldn't represent some posible values for the science fold position, and added a `Parked` option that didn't make sense for the requested light path.